data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,533 @@
+
+
+
+mission "Hai Legacy 1"
+	invisible
+	shipyard
+	source "Hai-home"
+	destination "Firelode"
+	clearance
+	to offer
+		has "First Contact: Unfettered: offered"
+		# note: this mission chain could be tweaked later to trigger even after they are friendly
+		"reputation: Hai (Unfettered)" < 0
+		not "Unfettered Jump Drive 1: offered"
+		# it will offer but defer itself the first time if you lack money but have sufficient net worth
+		"credits" + "net worth" * ( 1 - "Hai Legacy 1: deferred" ) >= 10000000
+	on offer
+		conversation
+			branch first
+				not "Hai Legacy 1: deferred"
+			`Do you want to head to the coordinates as the hooded figure suggested?`
+			choice
+				`	(Sure.)`
+					goto go
+				`	(Maybe another time.)`
+					defer
+				`	(Never.)`
+					decline
+			label first
+			`As you're browsing the shipyard, you are discreetly approached by a hooded figure. When you stop to see what they want to say, they grab and pull your arm to follow them. "Keep walking," the stranger hisses. After pointing at and commenting on random ship, as if to mimic the actions of regular passerby, they address you again. "My associates and I have a lucrative business proposition for you. If you are interested, come find us at these coordinates." Without waiting for a response, they fold a piece of paper into your hand before quickly fading into the crowd. It lists only a location, that you cross-reference with the local maps: a beach on the other side of the planet.`
+			`	However, it also mentions you need to have at least 10 million in cash! It may be a good idea to save up or take a debt and then come back another day to head to the location.`
+				to display
+					"credits" < 10000000
+				defer
+			`	Will you go there?`
+			choice
+				`	(Sure.)`
+				`	(Maybe another time.)`
+					defer
+			label go
+			`	While you're pondering whether there will be enough space to land your ship at the destination, you notice a train ticket was placed in your pocket, with instructions written on the back. You proceed to take the train and hike a few hours to get the rest of the way there.`
+			`	Upon arriving, you take a look around, looking for the stranger. Except for a cave, some sparse vegetation, and of course the beach itself, you cannot see much of interest in the vicinity. You double check that these are the right coordinates and look around some more.`
+			`	After waiting for some time, you spot two hooded figures further down the beach making their way toward you. You start to doubt whether or not this was a good idea, but it's too late to back out now. Even when they're close enough to speak, their faces somehow remain impossible to glimpse beneath the hoods. "Hello, <last>," one says to you. "We are sorry for all the complications, but the truth is, we cannot afford to take risks. It is best for all if you do not know who we are, and we needed a secluded area to speak to you."`
+			choice
+				`	"Tell me more."`
+					goto tellme
```

Comment:
```suggestion
					goto "tell me"
```
I know there's been some discussion on how to do multiword gotos, but I think it should be more than a cut space.